diesel/src/associations/mod.rs

```diff
@@ -146,10 +146,11 @@
 //! a user and all of its posts, that type is `(User, Vec<Post>)`.
 //!
 //! Next lets look at how to load the children for more than one parent record.
-//! [`belonging_to`][belonging-to] can be used to load the data, but we'll also need to group it
-//! with its parents. For this we use an additional method [`grouped_by`][grouped-by]
+//! [`belonging_to`] can be used to load the data, but we'll also need to group it
+//! with its parents. For this we use an additional method [`grouped_by`].
 //!
-//! [grouped-by]: trait.GroupedBy.html#tymethod.grouped_by
+//! [`grouped_by`]: trait.groupedby.html#tymethod.grouped_by
```

Comment:
At least for local docs on linux this does not work. Needs to be

```suggestion
//! [`grouped_by`]: trait.GroupedBy.html#tymethod.grouped_by
```